app/tests/content/test_event_integration.py

```diff
@@ -280,6 +280,23 @@ def test_create_event_as_admin(permission_test_util):
     )
 
 
+@pytest.mark.django_db
+def test_create_event_with_group_priorities(api_client, admin_user):
+
+    client = api_client(user=admin_user)
+    data = get_event_data()
+
+    groups = GroupFactory.create_batch(2)
+
+    groups = [{"slug": group.slug} for group in groups]
+
+    data["priority_pools"] = [{"groups": groups}]
+
+    response = client.post(API_EVENTS_BASE_URL, data)
+
+    assert response.status_code == 201
+
```

Comment:
Kunne du lagt til en assert her på at priority_pools'ene faktisk blir laget og dermed kommer med tilbake i serializeren/json()?
```suggestion
assert response.json()["priority_pools"] == ...
```

Reply:
<img width="334" alt="image" src="https://user-images.githubusercontent.com/31009729/157054462-f37b8844-b3dc-485e-8941-0b3efa501b82.png">

@Eirsteir en slik test kunne vært nice ^